### ISSUE TITLE
[BUG] Warning: Error in <Anonymous>: unable to find an inherited method for function '.multiSelectionResponsive' for signature '"CustomPlot"'

### ISSUE THREAD

**Issue by lee-t** [iSEE/iSEE#612]
**Describe the bug**
I'm creating a custom panel for iSEE using the function 'createCustomPlot'. This custom panel only generates a ggplot object and receives column selections from a single table, it will not need to transmit to other panels. 
I've encountered an error which crashes the shiny app which seems to be related to inherited methods written after the function was implemented. Related to #608 

**Minimal Reproduction**
Providing my own code might not be possible publicly. However, I can reproduce the error using the example found in the iSEE [vignette for custom panels](http://bioconductor.org/packages/devel/bioc/vignettes/iSEE/inst/doc/custom.html#23_Example_of_custom_table_panel). I've included the full app as a gist (https://gist.github.com/lee-t/2773791fe65a4da5c5604b4d03cd0e6d).

**Steps to Reproduce**
1. Run app.R
2. click or make a selection in the Violin plot "Column data plot1"
3. App will crash and print the following in the terminal:

```
Warning: Error in <Anonymous>: unable to find an inherited method for function ‘.multiSelectionResponsive’ for signature ‘"customPanel"’
  3: runApp
  2: print.shiny.appobj
  1: <Anonymous>
```
**Other Notes**
I have tried naively defining methods in [createCustomPanels.R](https://github.com/iSEE/iSEE/blob/master/R/createCustomPanels.R) to no avail. Doesn't seem to inherit anything from the other panel classes so I'm unsure why or when this error appeared in the package. 

**Session info**
```
> sessionInfo()
R version 4.2.1 (2022-06-23 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 22000)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] scRNAseq_2.10.0             scater_1.24.0               scuttle_1.6.3              
 [4] ggplot2_3.3.6               png_0.1-7                   aws.s3_0.3.21              
 [7] ggsci_2.9                   HDF5Array_1.24.2            rhdf5_2.40.0               
[10] DelayedArray_0.22.0         Matrix_1.5-1                shiny_1.7.2                
[13] iSEE_2.9.11                 SingleCellExperiment_1.18.1 SummarizedExperiment_1.26.1
[16] Biobase_2.56.0              GenomicRanges_1.48.0        GenomeInfoDb_1.32.4        
[19] IRanges_2.30.1              S4Vectors_0.34.0            BiocGenerics_0.42.0        
[22] MatrixGenerics_1.8.1        matrixStats_0.62.0         

loaded via a namespace (and not attached):
  [1] utf8_1.2.2                    shinydashboard_0.7.2          tidyselect_1.2.0             
  [4] RSQLite_2.2.18                AnnotationDbi_1.58.0          htmlwidgets_1.5.4            
  [7] BiocParallel_1.30.3           Rtsne_0.16                    aws.signature_0.6.0          
 [10] munsell_0.5.0                 ScaledMatrix_1.4.1            codetools_0.2-18             
 [13] ragg_1.2.3                    statmod_1.4.37                scran_1.24.1                 
 [16] DT_0.25                       miniUI_0.1.1.1                withr_2.5.0                  
 [19] colorspace_2.0-3              filelock_1.0.2                knitr_1.40                   
 [22] rstudioapi_0.14               shinyWidgets_0.7.4            labeling_0.4.2               
 [25] GenomeInfoDbData_1.2.8        bit64_4.0.5                   farver_2.1.1                 
 [28] vctrs_0.4.2                   generics_0.1.3                xfun_0.33                    
 [31] BiocFileCache_2.4.0           R6_2.5.1                      doParallel_1.0.17            
 [34] ggbeeswarm_0.6.0              clue_0.3-61                   rsvd_1.0.5                   
 [37] locfit_1.5-9.6                AnnotationFilter_1.20.0       bitops_1.0-7                 
 [40] rhdf5filters_1.8.0            cachem_1.0.6                  shinyAce_0.4.2               
 [43] assertthat_0.2.1              BiocIO_1.6.0                  promises_1.2.0.1             
 [46] scales_1.2.1                  beeswarm_0.4.0                gtable_0.3.1                 
 [49] beachmat_2.12.0               ensembldb_2.20.2              rlang_1.0.6                  
 [52] systemfonts_1.0.4             GlobalOptions_0.1.2           splines_4.2.1                
 [55] lazyeval_0.2.2                rtracklayer_1.56.1            BiocManager_1.30.19          
 [58] yaml_2.3.5                    GenomicFeatures_1.48.4        crosstalk_1.2.0              
 [61] httpuv_1.6.6                  rsconnect_0.8.27              tools_4.2.1                  
 [64] ellipsis_0.3.2                jquerylib_0.1.4               RColorBrewer_1.1-3           
 [67] Rcpp_1.0.9                    base64enc_0.1-3               sparseMatrixStats_1.8.0      
 [70] progress_1.2.2                zlibbioc_1.42.0               purrr_0.3.5                  
 [73] RCurl_1.98-1.9                prettyunits_1.1.1             GetoptLong_1.0.5             
 [76] viridis_0.6.2                 cowplot_1.1.1                 fontawesome_0.3.0            
 [79] ggrepel_0.9.1                 cluster_2.1.4                 magrittr_2.0.3               
 [82] circlize_0.4.15               colourpicker_1.1.1            ProtGenerics_1.28.0          
 [85] hms_1.1.2                     shinyjs_2.1.0                 mime_0.12                    
 [88] evaluate_0.17                 xtable_1.8-4                  XML_3.99-0.11                
 [91] gridExtra_2.3                 shape_1.4.6                   compiler_4.2.1               
 [94] biomaRt_2.52.0                tibble_3.1.8                  crayon_1.5.2                 
 [97] htmltools_0.5.3               mgcv_1.8-40                   later_1.3.0                  
[100] DBI_1.1.3                     ExperimentHub_2.4.0           dbplyr_2.2.1                 
[103] ComplexHeatmap_2.12.1         rappdirs_0.3.3                cli_3.3.0                    
[106] parallel_4.2.1                metapod_1.4.0                 igraph_1.3.5                 
[109] pkgconfig_2.0.3               GenomicAlignments_1.32.1      xml2_1.3.3                   
[112] foreach_1.5.2                 vipor_0.4.5                   bslib_0.4.0                  
[115] dqrng_0.3.0                   XVector_0.36.0                stringr_1.4.1                
[118] digest_0.6.29                 Biostrings_2.64.1             rmarkdown_2.17               
[121] rintrojs_0.3.2                edgeR_3.38.4                  DelayedMatrixStats_1.18.1    
[124] restfulr_0.0.15               curl_4.3.3                    Rsamtools_2.12.0             
[127] rjson_0.2.21                  lifecycle_1.0.3               nlme_3.1-160                 
[130] jsonlite_1.8.2                Rhdf5lib_1.18.2               BiocNeighbors_1.14.0         
[133] viridisLite_0.4.1             limma_3.52.4                  fansi_1.0.3                  
[136] pillar_1.8.1                  lattice_0.20-45               KEGGREST_1.36.3              
[139] fastmap_1.1.0                 httr_1.4.4                    interactiveDisplayBase_1.34.0
[142] glue_1.6.2                    iterators_1.0.14              bluster_1.6.0                
[145] BiocVersion_3.15.2            bit_4.0.4                     stringi_1.7.8                
[148] sass_0.4.2                    blob_1.2.3                    textshaping_0.3.6            
[151] BiocSingular_1.12.0           AnnotationHub_3.4.0           memoise_2.0.1                
[154] dplyr_1.0.10                  irlba_2.3.5.1    
```

**Comment by lee-t** [comment]
Thanks for getting back to this issue! 

So I've added your suggested code to the [script](https://gist.github.com/lee-t/2773791fe65a4da5c5604b4d03cd0e6d) but I don't think it's changed anything (added it at line#80).  Should I have put it at that part of the script or somewhere else?

**Comment by kevinrue** [comment]
OK. Sorry for the disappointment. That was supposed to work in my mind, but still a bit of a shot in the dark, and I hadn't tested it myself first. I'll look into it ASAP.

**Comment by kevinrue** [comment]
See https://github.com/iSEE/iSEE/issues/608#issuecomment-1412290364

**Comment by kevinrue** [comment]
Fixed in #617 

So sorry: had the PR ready back in February, personal events distracted me and I forgot to merge. Done now!
